moorhuhn-service/src/main/java/com/moorhuhnservice/moorhuhnservice/repositories/QuestionRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.moorhuhnservice.moorhuhnservice.repositories;
+
+import com.moorhuhnservice.moorhuhnservice.BaseClasses.Question;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface QuestionRepository extends CrudRepository<Question, Long>{

```

Comment:
I recommend `JpaRepository` which has simply more options than `CrudRepository`.

Reply:
I know crudrepository from sopra, it was allways more the enough. do you have a special example what jpa can do and crud cant?

Reply:
Jpa implements Crud and simply offers more functionality.
That's what I meant.

Reply:
changed it